httpd/handler.go

```diff
@@ -174,11 +179,110 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 		return
 	}
 
-	// Execute query. One result will return for each statement.
-	results := h.server.ExecuteQuery(query, db, user)
+	// get the chunking settings
+	chunked := q.Get("chunked") == "true"
```

Comment:
I thought chunking was requested using a special HTTP header? Is that not so? Are we talking about different "chunking"?